lib/git.js

```diff
@@ -11,7 +11,8 @@ var FORMAT         = '%H%n%s%n%b%n' + SEPARATOR;
  * Get all commits from the last tag (or the first commit if no tags).
  * @returns {Promise<Array<Object>>} array of parsed commit objects
  */
-exports.getCommits = function () {
+exports.getCommits = function (options) {
+  options = options || {};
```

Comment:
Good call here. Can you actually add this in [`package.js`](https://github.com/lob/generate-changelog/blob/21a80033a13f90ddee8c87479224c46dd2ca0572/lib/package.js#L51-L52) too? Then you can omit the empty object arg in your `version untouched` test.